qiskit/circuit/instruction.py

```diff
@@ -65,6 +75,7 @@ def __init__(self, name, num_qubits, num_clbits, params, duration=None, unit="dt
             duration (int or float): instruction's duration. it must be integer if ``unit`` is 'dt'
             unit (str): time unit of duration
             label (str or None): An optional label for identifying the instruction.
+            condition (tupple[ClassicalRegister, int] or None: classical condition of instruction.
```

Comment:
```suggestion
            condition (tuple[ClassicalRegister, int] or None: classical condition of instruction.
```

Reply:
Also, is `Instruction` not allowed to take a `Clbit` as its condition?

Reply:
That should be allowed.